ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -59,7 +59,7 @@ module('Acceptance | Enterprise | replication', function(hooks) {
     assert.equal(
       find('[data-test-empty-state-message]').textContent.trim(),
       'This cluster has not been enabled as a Disaster Recovery Secondary. You can do so by enabling replication and adding a secondary from the Disaster Recovery Primary.',
-      'renders default message specific to when no replication is enabled'
+      'renders default message specific to when no replication is disabled'
```

Comment:
i think the previous message was correct -- this is testing for when "no replication is enabled" (aka "when replication is disabled"), right?

Reply:
This got me at first, but I think what's going on, is on row 43 and 44 inside the hooks, the replications are disabled.  Then it goes on to test enabling each of them.  row 68 is where it begins to enable performance replication.

Reply:
ah yes, i understand that -- but i think the assertion message should be `renders default message specific to when no replication is enabled`, since replication has not been enabled yet? i.e. replace `when no replication is disabled` --> `when no replication is enabled`? or a clearer message `when replication is not enabled`? whichever you prefer 🤷‍♀️

Reply:
Yep makes total sense.  I'll amend.